src/main/resources/templates/tutors/create.html

```diff
@@ -1,45 +1,39 @@
-        <link rel="stylesheet" href="../css/shards.min.css">
-    </head>
-    <body>
-        <div class="container my-5">
-            <h2 class="mb-5">New Tutor</h2>
-            <div class="row">
-                <div class="col-md-6">
-                    <form action="#" th:action="@{/tutors/add}" th:object="${tutor}" method="post">
-                        <div class="row">
-                            <div class="form-group col-md-6">
-                                <label for="fname" class="col-form-label">First Name</label>
-                                <input type="text" th:field="*{fname}" class="form-control" id="fname" placeholder="First Name">
-                                <span th:if="${#fields.hasErrors('fname')}" th:errors="*{fname}" class="text-danger"></span>
-                            </div>
-                            <div class="form-group col-md-6">
-                                <label for="lname" class="col-form-label">Last Name</label>
-                                <input type="text" th:field="*{lname}" class="form-control" id="lname" placeholder="Last Name">
-                                <span th:if="${#fields.hasErrors('lname')}" th:errors="*{lname}" class="text-danger"></span>
-                            </div>
-                            <div class="form-group col-md-6">
-                                <label for="email" class="col-form-label">Email</label>
-                                <input type="text" th:field="*{email}" class="form-control" id="email" placeholder="Email">
-                                <span th:if="${#fields.hasErrors('email')}" th:errors="*{email}" class="text-danger"></span>
-                            </div>
-                        </div>
-                        <div class="row">
-                            <div class="col-md-6 mt-5">
-                                <input type="submit" class="btn btn-primary" value="Add Tutor">
-                            </div>
-                        </div>    
-                    </form>
-                </div>
-            </div>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <title>Add Tutor</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.4.1/css/all.css"
+      integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../css/shards.min.css" />
+  </head>
+  <body>
+    <div class="container my-5">
+      <h2 class="mb-5">New Tutor</h2>
+      <div class="row">
```

Comment:
Yinon and I are reviewing this and are not quite sure what this is doing. Are you just abstracting this to src/main/resources/templates/tutors/form.html?

Reply:
Your in person response was helpful, thank you!